flake.nix

```diff
@@ -67,6 +80,9 @@
             hardeningDisable = [ "fortify" ];
 
             shellHook = ''
+              # install required CF CLI plugins
+              cf install-plugin "$(whereis -q app-autoscaler-cli-plugin)" -f
```

Comment:
We could even get rid of this. There is a standard pattern for defining nix-packages that let's you easily bundle. But perhaps that may be overkill for now as the nixpkgs-upstream-cf-cli is still not built for that.